cmake/morpheus_utils/python/python_module_tools.cmake

```diff
@@ -59,13 +79,20 @@ function(morpheus_utils_create_python_package PACKAGE_NAME)
   file(GLOB_RECURSE wheel_python_files
     LIST_DIRECTORIES FALSE
     CONFIGURE_DEPENDS
-    "*.py"
-    "py.typed"
-    "pyproject.toml"
-    "setup.cfg"
-    "MANIFEST.in"
+    "${src_dir}/.*.py"
+  )
+
+  file(GLOB wheel_python_project_files
+      LIST_DIRECTORIES FALSE
+      CONFIGURE_DEPENDS
+      "${project_dir}/py.typed"
```

Comment:
Looks like this is what we were doing for morpheus:
```
# Get all of the python files to copy to the build directory
file(GLOB_RECURSE MORPHEUS_PYTHON_FILES
  RELATIVE ${PROJECT_SOURCE_DIR}
  LIST_DIRECTORIES FALSE
  CONFIGURE_DEPENDS
  "morpheus/*.py"
  "morpheus/py.typed"
)

set(MORPHEUS_ROOT_PYTHON_FILES
  "pyproject.toml"
  "setup.cfg"
  "setup.py"
  "versioneer.py"
  "MANIFEST.in"
)
```

So we need a couple of changes:
- `py.typed` should be up in the `${src_dir}` glob.
- Need to add `setup.py`
- Need to add `versioneer.py`

Reply:
Updated.